Gaydachuk_Basket_Example.js

Matthew-Gaydachuk opened the pull request:
Взял пример куска кода из одного из своих старых проектов. Товары было бы все же хорошо отдельно в какой-то файл засунуть, к примеру goods.json

Matthew-Gaydachuk committed:
Gaydachuk_Basket_Example.js

Взял пример куска кода из одного из своих старых проектов. Товары было бы все же хорошо отдельно в какой-то файл засунуть, к примеру goods.json

s-lozhechnikov commented on Lab1/Basket.js:
```diff
@@ -1,34 +1,36 @@
-	constructor(product) {
-		//implement
-	}
+$('document').ready(function(){
```
У нас подключен жКуери?

s-lozhechnikov commented on Lab1/Basket.js:
```diff
@@ -1,34 +1,36 @@
+    loadGoods();
+});
+
+function loadGoods() {
```
Глобальная функция

abyrvalg commented on Lab1/Basket.js:
```diff
@@ -1,34 +1,36 @@
 
+function addToBasket() {
+    //download goods to my basket
+    var articul = $(this).attr('data-art');
```
What is this?

s-lozhechnikov commented on Lab1/Basket.js:
```diff
@@ -1,34 +1,36 @@
+            out+='</div>';
+        }
+        $('#goods').html(out);
+        $('button.add-to-basket').on('click', addToBasket);
```
Насколько помню у нас нет кнопок и хтмл вообще

Matthew-Gaydachuk committed:
Gaydachuk_Example_Basket_2.0

Кнопки и html убрал... Хотя это и нужно для связания с основным html... Как просто без кнопок добавлять элементы в корзину... Хотя мы учим native js, в общем, не суть... Jquery подключил. А глобальная функция там вроде как и должна быть...
